custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -49,6 +49,9 @@ class WorkAnIssue(
     }
 
     private fun edit(issuePage: AbstractIssuePage) {
```

Comment:
In order for this to be used, we gotta bump `editProbability` in `ScenarioSimilarities`